api/task.go

```diff
@@ -55,10 +51,11 @@ func (h TaskHandler) AddRoutes(e *gin.Engine) {
 	routeGroup.POST(TasksRoot, h.Create)
 	routeGroup.GET(TaskRoot, h.Get)
 	routeGroup.PUT(TaskRoot, h.Update)
+	routeGroup.PATCH(TaskRoot, Transaction, h.Patch)
 	routeGroup.DELETE(TaskRoot, h.Delete)
 	routeGroup.GET(TasksReportQueueRoot, h.Queued)
 	// Actions
-	routeGroup.PUT(TaskSubmitRoot, h.Submit, h.Update)
+	routeGroup.PUT(TaskSubmitRoot, Transaction, h.Submit)
```

Comment:
hint: with complexity delegated to the task manager, simpler to implement submit as a patch and no longer chain submit; modBody(); update().
BaseHandler.modBody() should probably go away.